scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -103,6 +105,53 @@ static __always_inline struct layer *lookup_layer(u32 id)
 	return &layers[id];
 }
 
+static __always_inline 
+struct bpf_cpumask *idle_cpumask_unprotected(struct cpu_ctx *cpuc)
+{
+	struct bpf_cpumask *cpumask;
+	const struct cpumask __kptr *idlemask;
+
+	if (!(cpumask = bpf_cpumask_create()))
```

Comment:
I don't think we want to hot-allocate the cpumask on the spot. We already use cached masks extensively when picking idle CPU. It'd be better if we integrate this mask there instead of layering it on top of `scx_bpf_get_idle_cpumask()`.